### DISCUSSION TITLE
Streaming to bluetooth speaker

### DISCUSSION THREAD

**Discussion by jds11111** [sle118/squeezelite-esp32#258] [Q&A] [answered]
Has anyone gotten this to work with an ESP32 WROVER board? I have been playing with it a few hours with two different boards.
The board can see three BT objects to pair with, one of which is my amp.  However, at this point, the web UI starts reporting lots of timeouts, and I am usually unable to choose the amp.  I did get it to work for a few minutes, but after a bit it stopped, and have been unsuccessful since. 

I suspect that it might have to do with the bluetooth and WIFI radios sharing the antenna, but just guessing.  Has anyone gotten this to work that can remember what the tricks were?  I am happy to test and report whatever helps.

**Comment by daduke** [comment]
yes I use about 5 WROVERs this way. Very stable, but I'm on build 1180 since newer ones don't reliably connect to wifi for me.

**Reply by jds11111** [reply]
Thanks for the tip. However, when I flash the 1180 firmware (either using esptool or the esphome web flasher), neither of my two chips broadcast an AP hotspot to connect to.  How did you flash the older firmware?

**Comment by jds11111** [comment, marked as answer]
I got help in a (now-closed) issue: https://github.com/sle118/squeezelite-esp32/issues/257
